### ISSUE TITLE
Make Data Subject Identification Consistent with PRIV

### ISSUE THREAD

**Issue by milstan** [blindnet-io/product-management#1002]
### naming consistency
PRIV defines a data-subject identity through 2 data elements:
- `dsid`
- `dsid-schema`
See details in [PRIV specification](https://github.com/blindnet-io/product-management/blob/main/refs/schemas/priv/RFC-PRIV.md#globally-unique-data-subject-identities).

In several parts of the DevKit we use wording that is not consistent with this, and I think we'd gain by hamronizing and having the same wording everywhere. PCE uses [id and schema](https://github.com/blindnet-io/privacy-computation-engine/blob/develop/modules/priv/src/main/scala/DataSubject.scala). Same is with the [Demo application and the demo server](https://github.com/blindnet-io/privacy-components-web/blob/main/demos/devkit-simple-tutorial/src/views/Form.js).

### e-mail schema interpretation cosnsistecy
In addition to the miscmatch of naming, there is a worse proble. PRIV defines few [DSID schemas](https://github.com/blindnet-io/product-management/blob/main/refs/schemas/priv/RFC-PRIV.md#data-subject-id-schemas), none of which is called `dsid`.

The demo and the demo server use this value 
```
schema: 'dsid'
```
And as the value of the corresponding `id` parameter, they accept the raw e-mail of the user.

The only data subject id schema that comes close, that we have in PRIV, is `email-sha-256` which accepts as `dsid` values the hashed e-mail, and as `dsid-schema` : `email-sha-256`.

## Whant we want
We want PRIV and Devkit to be in sync => we must change one or the other (or both).

Regarding naming I am fine with changes either way, as long as we have a motive.

Regarding the absence of the schema using raw e-mail, I viable case might be made for adding one in PRIV(at least for the purposes of testing)

**Comment by TheKinrar** [comment]
The demo backend actually only knows about email addresses. This makes sense though, as it represents a third-party app that does not use PRIV for storing its data.

I believe that right now we basically only use the email addresses as an ID and don't have a concept of schema implemented. @m4rk055 can probably confirm this for the PCE side. At least that's how authentication currently works.

**Comment by m4rk055** [comment]
Yes, schemas are only stored together with the user's id and carry no semantic.
